Update with 0.10.0 changes (#532)
chocolatey/docs: input/en-us/central-management/usage/website/deployments.md

        ![CCM deployment schedule picker](/assets/images/deployments/ccm-deployments-set-schedule-datetime.png)

    * (Optional) If you'd like to define a maintenance window for the deployment start time, select the **Restrict schedule to a maintenance window** option and enter the ending date and time for the maintenance window.

       ![CCM deployment maintenance window option](/assets/images/deployments/ccm-deployments-maintenance-window.png)

    * (Optional) If you'd like a deployment to happen again, on a recurring basis. Check the [recurring deployments section for more information](#recurring-deployments)

        ![CCM deployment Repeat Period](/assets/images/deployments/ccm-deployments-repeat-period.png)

1. Select :heavy_plus_sign: **Add Step** to add your first deployment step.

    ![CCM deployment add step button](/assets/images/deployments/ccm-deployments-add-step.png)

1. (Optional) In the `Create New Deployment Step` modal, enter a custom name for the deployment step.

    ![CCM deployment new step modal](/assets/images/deployments/ccm-deployments-new-step-modal.png)

1. Add the deployment step action:

>    * For _Basic_ deployment steps, select a `Script command` from the list, a `Package name` to install, and optionally a specific package version to install, or whether to use the `--pre` option, i.e. to install the latest prerelease package version.  **NOTE:** It is not possible to use a space character within the `Package name` or `Package Version` textboxes, and the `Package Version` textbox must contain at least one digit.

Reviewer: ```suggestion
   * For _Basic_ deployment steps, select a `Script command` from the list, a `Package name` to install, and optionally a specific package version to install or whether to allow Chocolatey to install the latest prerelease package version.  **NOTE:** It is not possible to use a space character within the `Package name` or `Package Version` textboxes, and the `Package Version` textbox must contain at least one digit.
```

My understanding is that often abbreviations like `i.e.` have inconsistent at best readability for some screen readers, and can also be unfamiliar for those learning English as a second language. Best to avoid them where possible, IMO.

Also, I'm not sure it's particularly relevant to mention the individual Chocolatey CLI command option in this instance. It's related, sure, but IMO Central Management's feature sets don't need to call out the individual CLI features they happen to be using. (We don't mention `--version` the same way, either.)

See also: https://www.oit.va.gov/accessibility/getting-started/abbreviations

    ![CCM deployment step Select Target Groups modal](/assets/images/deployments/ccm-deployments-step-select-groups-modal.png)

1. Click the :floppy_disk: **Save** button to save the step.

    ![CCM deployment step Save button](/assets/images/deployments/ccm-deployments-step-save.png)

1. Continue to add steps until your deployment is complete.
1. Select :floppy_disk: **Save** to save the changes to the deployment.

## Recurring Deployments

As mentioned above, when creating a deployment it possible to select a scheduled start date/time, and in addition a Repeat Period.  This Repeat Period controls how often a deployment recurs going forward. The values for the Repeat Period are:

- `Daily`
- `Weekly`
- `Every two weeks`
- `Every four weeks`
- `Monthly`
- `Every two months`
- `Quarterly`
- `Every six months`
- `Yearly`

Once a deployment has been assigned a Repeat Period, and it is moved to the [Ready](#ready) state, it will be shown with a slightly different icon:

![CCM deployment marked as recurring](/assets/images/deployments/ccm-deployments-recurring-icon.png)

> Let's take as example a deployment in the [Ready](#ready) state that is scheduled to start on `23rd August 2022 at 07:11 UTC`, and a Repeat Period of `Weekly` has been chosen.  Once this instance of the deployment moves to the [Active](#active) state, another instance of the deployment (with the same steps and settings will be created), with the only different being the scheduled start date/time, which will be set to exactly 1 week from the scheduled start date/time of the previous instance, in this case `30th August 2022 at 07:11 UTC`.  A new instance of the deployment will be created once a scheduled deployment moves to the [Active](#active) state. In addition to the schedule start date/time, if a deployment contains a `Last Schedule Date Time`, this will also be set to exactly 1 week from the `Last Scheduled Date Time` of the previous instance of the deployment.

Reviewer: Not really a fan of giant chunks on one line like this, it makes it difficult to diff later on (and review!). 😕

We're also putting general information in the same paragraph as the specific example, which I think is confusing.

I think we could use less commas and more distinct sentences to make it a little easier to follow as well.

```suggestion
Let's take as example a deployment in the [Ready](#ready) state that is scheduled to start on `23rd August 2022 at 07:11 UTC`, with a Repeat Period of `Weekly` set.
Once this instance of the deployment moves to the [Active](#active) state, another instance of the deployment will be created.
The new instance will have the same steps and settings except for the scheduled start date/time, which will be set to exactly 1 week from the scheduled start date/time of the previous instance (in this case `30th August 2022 at 07:11 UTC`).

For repeating deployments, a new instance of the deployment will be created once a scheduled deployment moves to the [Active](#active) state.
If a repeating deployment specifies a maintenance window date/time (`Last Scheduled Date Time`), the new instance's maintenance window will also be adjusted from the previous instance's value by the same period as the scheduled start date/time.
```

> > :warning: WARNING
> >
> > One important thing to remember is that if a scheduled start date/time of a deployment is overridden using the [Run Now](#run-now) action, the scheduled date/time of the new instance of the recurring deployment will continue to use the scheduled start date/time of the previous instance when creating the new instance of the recurring deployment, **not** the date/time that the deployment actually started.  If you want/need to change the scheduled start/date of the recurring deployment edit the deployment while in the [Ready](#ready) state, and future instances of the recurring deployment will use this new value.

Reviewer: ```suggestion
> :warning: WARNING
>
> If the scheduled start date/time of a deployment is overridden using the [Run Now](#run-now) action, the new instance of the recurring deployment will use the **scheduled** start date/time of the previous instance when calculating the next scheduled start date/time, **not** the date/time that the deployment actually started.
> If you want to change the scheduled start date/time of the recurring deployment, edit the deployment while it is in the [Ready](#ready) state to ensure that future instances of the recurring deployment will use that value when calculating the next scheduled date/time.
```

While in the [Ready](#ready) state, if you use the [Cancel](#cancel) or [Delete](#delete) action on the recurring deployment, no further instances of the recurring deployment will be created.

## Deployment States

### Draft

A deployment is initially created in the [`Draft`](#draft) state, and will remain in this state until it is moved into the [`Ready`](#ready) state.
While it is in the [`Draft`](#draft) state, it cannot be run, and scheduled deployment start times will be ignored.

While in the [`Draft`](#draft) state, the available actions that can be performed on a deployment plan are:

- [Move To Ready](#move-to-ready)
- [Edit](#edit)
- [Duplicate](#duplicate)
- [Delete](#delete)

### Ready

Once the deployment enters the [`Ready`](#ready) state, it's eligible to be started.
Deployments in this state can be started manually or according to a schedule.

> :memo: **NOTE**
>

> > Any further modifications to a deployment in this state will revert it back to the [`Draft`](#draft) state.

Reviewer: ```suggestion
> Any modifications to a deployment in this state will revert it back to the [`Draft`](#draft) state.
```

While in the [`Ready`](#ready) state, the available actions that can be performed on a deployment plan are:

- [View](#view)
- [Run Now](#run-now)
- [Edit](#edit)
- [Duplicate](#duplicate)
- [Cancel](#cancel)
- [Delete](#delete)

### Active

Deployments that are currently in progress will be in this state.

While in the [`Active`](#active) state, the available actions that can be performed on a deployment plan are:

- [Details](#details)
- [Duplicate](#duplicate)
- [Cancel](#cancel)

### Completed

Deployments that have completed running will be in either the `Success`,  `Failed`, `Unreachable`, `Inconclusive`, or `Cancelled` state, depending on how the run went.

While in one of these states, the available actions that can be performed on a deployment plan are:

- [Details](#details)
- [Duplicate](#duplicate)
- [Archive](#archive)

### Archived

Deployments that are in a completed state can be actioned using [`Archive`](#archive) action to hide them from the main Deployments screen.
This is helpful if you'd like to reduce clutter on the main deployments screen without discarding the information the completed report contains.

> You can access archived deployments from the `Deployments` page and clicking on the `View Archived Deployments` button.  [`Archived](#archived) deployments will not appear in any other reports that contain deployment plans.

Reviewer: ```suggestion
You can access archived deployments from the `Deployments` page and clicking on the `View Archived Deployments` button.  [`Archived`](#archived) deployments will not appear in any other reports that contain deployment plans.
```

While in the [`Archived`](#archived) state, no additional actions can be performed on a deployment plan.

## Deployment Plan Actions

Depending on the [state](#deployment-states) that a deployment plan is currently in, there are a defined set of actions that can be performed on them.  What follows are is a description of each of those actions.

### Move To Ready

This action moves a deployment plan from the [`Draft`](#draft) state to the [`Ready`](#ready) state. While in this interim state, no additional changes can be made to the deployment plan.  If changes are made, it will be moved back to the [`Draft`](#draft) state.

### Edit

The action opens the edit page for the selected deployment plan.  Here changes can be made to the steps, schedule, groups, etc.  If any changes are made on the page, a deployment plan that was in the [`Ready`](#ready) state, will be moved back to the [`Draft`](#draft) state.

### Duplicate

> :memo: **NOTE**
>
> This feature became available starting with Chocolatey Central Management 0.10.0

> The action makes an exact copy (with the exception of any scheduled start/end time) of any given deployment plan.  Once the duplicate has been created, you will be taken to the edit page ready to make any additional required changes.  The name of the new deployment plan will be the same as the original with some additional information added to the end.  For example, if the original deployment plan was called `Install Papercut`, the name given to the duplicate would be `Install Papercut - Copy (19 Aug 2022 12:52:25)` where the date time is the current date time.

Reviewer: ```suggestion
The action makes an exact copy (with the exception of any scheduled start/end time or repeat period) of any given deployment plan.  Once the duplicate has been created, you will be taken to the edit page ready to make any additional required changes.  The name of the new deployment plan will be the same as the original with some additional information added to the end.  For example, if the original deployment plan was called `Install Papercut`, the name given to the duplicate would be `Install Papercut - Copy (19 Aug 2022 12:52:25)` where the timestamp is the current date time.
```

> :memo: **NOTE**
>

> > There is a specific permission to allow duplication of a deployment plan. If you want to configure this permission, it can be done either for an individual [user](xref:ccm-administration-users), or a specific [role](xref:ccm-administration-roles)

Reviewer: ```suggestion
> There is a specific permission to allow duplication of a deployment plan. If you want to configure this permission, it can be done either for an individual [user](xref:ccm-administration-users), or a specific [role](xref:ccm-administration-roles)

```

### Delete

This removes all information about the deployment plan from Chocolatey Central Management.

The action cannot be undone, so care should be taken before using it.

### View

The action opens the edit page for the selected deployment plan where all the parts of the plan can be viewed.  Here changes can be made to the steps, schedule, groups, etc.  If any changes are made on the page, a deployment plan that was in the [`Ready`](#ready) state, will be moved back to the [`Draft`](#draft) state.

### Run Now

This action takes a deployment plan from the [`Ready`](#ready) state, to the [`Active`](#active) state.  This can be thought of as actually setting the deployment plan in motion, and the steps within this deployment plan will begin to be picked up by the computers that are contained within the steps (in the order that has been defined).

> If a deployment plan is in the ready state, but is waiting for the scheduled start time before automatically starting, it is possible to override this start time using the `Run Now` action.  Care should be taken here though if the deployment plan is also a recurring deployment plan.  In this case, the overriding the scheduled start time will mean that the next deployment plan will be configured to use the scheduled start time for the next iteration, and _not_ the overridden start time.

Reviewer: ```suggestion
If a deployment plan is in the ready state, but is waiting for the scheduled start time before automatically starting, it is possible to override this start time using the `Run Now` action.  Care should be taken here though if the deployment plan is also a recurring deployment plan.  In this case, the overriding the scheduled start time will mean that the next deployment plan will be configured to use the scheduled start time for the next iteration, and _not_ the overridden start time.
```

Do we need to mention this again given it was called out in a warning above already? Should we copy the warning down to here rather than rephrasing the same warning and putting it in the main text?

Author: That is a good catch, I will get this fixed up, using the warning in both places.

### Cancel

This action stops all future steps from being performed within a deployment plan.  Any steps that are currently active will still be allowed to complete, but no further steps will occur.

> Care should be taken when choosing to cancel a scheduled deployment, which has also been setup to a be a recurring deployment.  If you cancel the non-active version of a recurring deployment, no new deployment plans will be created in this series, and a new recurring deployment plan will have to be created.

Reviewer: This reads a bit indirect and a little confusing to me.

```suggestion
If a recurring deployment is Cancelled from the [`Ready`](#ready) state, all future instances of the plan are also cancelled.
If you want to skip an iteration of a recurring deployment, instead change its scheduled start time directly.
```

I think this framing might be a little more clear?

### Details

> This action opens up the details for the selected deployment plans, allowing the user to see the progress so far.  i.e. which step is currently active, which computers have picked up this step, etc.  The screen can be very useful for monitoring the progress in realtime, as the details pages update automatically.

Reviewer: ```suggestion
This action opens up the details for the selected deployment plans, allowing the user to see the progress so far.  For example: the currently active step, or which computers have picked up this step.  This view is useful for monitoring the progress in real time, as the details pages update automatically.
```
